### PR TITLE
Hotfix/change python shebang

### PR DESCRIPTION
**Pull request by JacobAMason**
Avoids issues with computers having python3 as the default instead of python2

### CODE REVIEW

File: cr17/nodes/cr17/obstacle_mapper.py [hunk 1]
```diff
@@ -1,15 +1,38 @@
-#!/usr/bin/env python
+#!/usr/bin/env python22
```
Comment (bgrayd) [inline cr17/nodes/cr17/obstacle_mapper.py]: I don't think there is a python22 yet, unless it is the 2.2 version of python, in which case python needs to work on its naming convention.

**Review by bgrayd — CHANGES_REQUESTED** [review]

File: cr17/nodes/cr17/obstacle_mapper.py (hunk 1 above)
Comment (JacobAMason) [reply cr17/nodes/cr17/obstacle_mapper.py]: Kek. I branched off of obstacle-mapper instead of develop

Comment (bgrayd) [reply cr17/nodes/cr17/obstacle_mapper.py]: yea, I was wondering about that, but since the node runs fine I was going to leave it

Comment (JacobAMason) [reply cr17/nodes/cr17/obstacle_mapper.py]: Should be good now

**Review by bgrayd — APPROVED** [review]

**Pull request merged**